cypress.config.ts

```diff
@@ -1,9 +1,43 @@
 import { defineConfig } from "cypress";
+import LoginPage = require("./cypress/pages/LoginPage");
+
+let employeeId: string;
+type employee = {
+  id: string,
+  username: string,
+  password: string
+};
+let employeeInfo: employee = {
+  id: "",
+  username: "",
+  password: "",
```

Comment:
How are we handling getter methods with empty data?

Reply:
We don't use getters until we set values from the test cases, because I want data to be set when test run not to be static

Reply:
1. we should handle it